### DISCUSSION TITLE
How do I fix this error? Just merely trying to toy with a phyton file and getting this error

### DISCUSSION THREAD

**Discussion by nezam7** [mkaring/ConfuserEx#438] [Q&A] [answered]
[ERROR] Failed to load "bot.exe" - Assembly does not appear to be a .NET assembly: ".NET data directory RVA is 0".

![image](https://user-images.githubusercontent.com/4648255/142155069-18274e83-1774-4d57-b635-a44da6c1aa15.png)

**Comment by mkaring** [comment, marked as answer]
The assembly you are trying to process isn't a .NET assembly. What are you loading there?

**Comment by nezam7** [comment]
a program from phyton. a simple program for pixel detection

**Reply by ElektroKill** [reply]
Python files or executables compiled from python files are not .NET executables.
A .NET executable is created when compiling, for the most part, C#, VB.NET, or F#.

**Reply by nezam7** [reply]
thanks. no wonder it doesnt work!
